packages/terra-theme-context/src/ThemeContextProvider.jsx

```diff
@@ -8,9 +8,10 @@ const propTypes = {
    */
   children: PropTypes.element.isRequired,
   /**
-   * An object containing the className of the selected theme.
+   * An object containing the name and className of the selected theme.
    */
   theme: PropTypes.shape({
+    name: PropTypes.string,
```

Comment:
What will the default theme be defined as?

Reply:
Terra application base will define an undefined or empty string as 'terra-default-theme' when no theme is defined in theme config. That seemed to make sense to me.